src/main/java/com/bullhornsdk/data/model/entity/core/standard/JobOrder.java

```diff
@@ -18,8 +18,8 @@
 @JsonInclude(JsonInclude.Include.NON_NULL)
 @JsonRootName(value = "data")
 @JsonPropertyOrder({ "id", "address", "appointments", "approvedPlacements", "assignedUsers", "benefits", "billRateCategoryID",
-		"bonusPackage", "branch", "branchCode", "businessSectors", "categories", "certificationList", "certifications", "clientBillRate",
-		"clientContact", "clientCorporation", "correlatedCustomDate1", "correlatedCustomDate2", "correlatedCustomDate3",
+		"bonusPackage", "branch", "branchCode", "businessSectors", "categories", "certificationList", "certifications", "certificationsGroups",
```

Comment:
is certificationsGroups a typo? shouldn't it be certificationGroups

Reply:
Yeah typo on my part. Fixed